doc/manual/src/glossary.md

```diff
@@ -7,6 +7,59 @@
     translated into low-level *store derivations* (implicitly by
     `nix-env` and `nix-build`, or explicitly by `nix-instantiate`).
 
+    [derivation]: #gloss-derivation
+
+  - [store derivation]{#gloss-store-derivation}\
+    A [derivation] represented as a `.drv` file in the [store].
+    It has a [store path], like any [store object].
+
+    See [`nix show-derivation`](./command-ref/new-cli/nix3-show-derivation.md) (experimental) for displaying the contents of store derivations.
```

Comment:
These new links are inconsistent with others in this file.

E.g. on line 141, the link is `command-ref/conf-file.html#conf-substituters`:
* no prefix `./`
* ends in `.md` vs `.html`

Is there a reason to have different link syntaxes?

Reply:
Fixed.

Reply:
I can't resolve threads (thanks Github permissions), but this now looks good